src/content/docs/en/guides/view-transitions.mdx

```diff
@@ -235,17 +235,68 @@ const myFade = {
+- Clicks on `<a>` elements.
+- Backwards and forwards navigation events.
 
-```html
-<a href="/quarterly-earnings.pdf" data-astro-reload>
+The following sections discuss how you can further control when navigation occurs within the router.
+
+### Preventing client-side navigation
+
+There are some cases where you cannot navigate via CSR, since both pages involved must use the `<ViewTransitions />` router to prevent a full-page reload. You may also not want client-side routing on every navigation change, and would prefer a traditional page navigation on select routes instead.
+
+You can opt out of client-side routing on a per-link basis by adding the `data-astro-reload` attribute to any `<a>` tag. This attribute will override any existing `<ViewTransitions />` component and instead trigger a browser refresh during navigation.
+
+The following example shows preventing client-side routing when navigating to an article from the home page only. This still allows you to have animation on shared elements, such as a hero image, when navigating to the same page from an article listing page:
+
+```astro title="src/pages/index.astro"
+<a href="/articles/emperor-penguins" data-astro-reload>
+```
+
+```astro title="src/pages/articles.astro"
+<a href="/articles/emperor-penguins">
 ```
 
-These links will be ignored by the router and a full page navigation will occur.
+Links with the `data-astro-reload` attribute will be ignored by the router and a full page navigation will occur.
+
+### Trigger navigation
+
+You can also trigger client-side navigation via events not normally listened to by the `<ViewTransitions />` router using `navigate`. This method from the `astro:transitions/client` module is available in any client script or client component.
+
+The following example shows using an option selected by a site visitor to trigger navigation: 
+
+```js title="form.js" "navigate(href)"
+import { navigate } from 'astro:transitions/client';
+
+// Navigate to the selected option automatically.
+document.querySelector('select').onchange = (ev) => {
+  let href = ev.target.value;
+  navigate(href);
+};
+```
+
+The `navigate` method takes these arguments:
+
+- `href` - The new page to navigate to. This must be a URL to a page within your own domain. Any external links will trigger redirection.
+- `options` - An object with the following properties:
+    - `history` - Either `'push'`, `'replace'`, or `'auto'`. When the option is `'push'` the router will use `history.pushState` to create a new entry in the browser history. If the option is `'replace'` it will use `history.replaceState` which will update the URL without adding a new entry into navigation. The `'auto'` option will also do a `history.pushState` in most cases, but if the URL is not one that can be transitioned to, it will not throw and the current URL will remain with no changes to the browser history.
```

Comment:
We can make this sentence a bit less verbose.
```suggestion
    - `history` - Either `'push'`, `'replace'`, or `'auto'`. When the option is `'push'` the router will use `history.pushState` to create a new entry in the browser history. If the option is `'replace'` it will use `history.replaceState` which will update the URL without adding a new entry into navigation. The `'auto'` option will also do a `history.pushState` in most cases, but if the URL cannot be transitioned to, it will not throw and the current URL will remain with no changes to the browser history.
```

Reply:
Resolving this, since I've rewritten for readability! Thanks for noticing this could be improved, Yan!